Arelle/Arelle
# NL Conformance Suite

#### Reason for change
Resolves #903 

#### Description of change
Transfer NL validations implemented so far into conformance suite format.

#### Steps to Test
Not added to CI yet. Run each disclosure system locally:
- Add NL conformance suite configs to `ALL_CONFORMANCE_SUITE_CONFIGS`.
- Run `python ./tests/integration_tests/validation/run_conformance_suites.py --test --log-to-file --offline --download-missing --name nl_nt16,nl_nt17,nl_nt18` to run all three disclosure systems.
- Confirm all tests pass.


**review**:
@Arelle/arelle
